requirements.txt

```diff
@@ -5,4 +5,7 @@ termcolor==2.3.0
 flask==2.3.2
 flask_wtf==1.1.1
 wtforms==3.0.1
+pandas==1.5.3
```

Comment:
The versions of `pandas` and `numpy` you specified cause compatibility issues for me. I'd suggest to just remove the version specifications there.